web_src/css/repo.css

```diff
@@ -3083,46 +3083,86 @@ tbody.commit-list {
   }
 }
 
-.pr-status {
+.ui.segment.commit-status-list {
+  padding: 0;
+  max-height: 30vh;
+  overflow-x: hidden;
+  transition: max-height 0.4s;
+  border-bottom: 0 !important;
+}
+
+.pr-commit-status.hide {
+  border-bottom: 0 !important;
+}
+
+.pr-commit-status.hide .ui.attached.header {
+  border-bottom: 0 !important;
```

Comment:
`none` instead of `0`. I do wonder whether there are simpler solutions to all this border hiding.

Reply:
Done in https://github.com/go-gitea/gitea/pull/26284/commits/fef5fc1fde01c9a93a78fc08a4c706662755db8f